services/mozilla-observatory/mozilla-observatory.service.js

```diff
@@ -26,6 +26,10 @@ const schema = Joi.object({
     .required(),
 }).required()
 
+const queryParamSchema = Joi.object({
+  publish: Joi.equal(''),
```

Comment:
As above, this is how `?publish` is represented which is what the badge customizer creates for this badge. Have update the documentation to match.

Reply:
I'm more confused now around why/when this is necessary. Is it when we're expecting query params representing a boolean value?

Reply:
Also if anyone is using this badge and had explicitly added a query param with false (i suppose `?publish=false`) will the param maintain the false value/get dropped entirely in the validation? Just want to make sure that wouldn't somehow get converted to `?publish` and have the reverse behavior

Reply:
> I'm more confused now around why/when this is necessary.

It's super confusing. `camp`, `querystring` and basically every other querystring library have different ideas about how to represent these values.

> Is it when we're expecting query params representing a boolean value?

I'm actually not sure whether Scoutcamp passes through the string `'false'` or converts it to a boolean. Since the schema here isn't converting anything, I think it would reject `?publish=false`.

Reply:
I've finished going through all the changes and ready to approve. I'd like run a quick test in my local env on this particular one just to triple check if that's cool with you

Reply:
Yea, of course!